bootstrap/src/main/java/org/jboss/windup/bootstrap/Bootstrap.java

```diff
@@ -258,31 +257,32 @@ else if ("--generateCompletionData".equals(arg))
             furnaceService.getFurnace().addRepository(AddonRepositoryMode.MUTABLE, new File(getUserWindupDir(), "addons"));
         }
 
+        if (command == null && updateRulesets && onlyRulesetsUpdateRequested(arguments))
```

Comment:
It seems like we probably need to be able to do both comments (e.g. allow multiple commands to be executed based on CLI values supplied.)